Detectors/EMCAL/calib/include/EMCALCalib/CalibDB.h

```diff
@@ -256,19 +257,33 @@ class CalibDB
   /// \throw TypeMismatchException if object is present but type is different (CCDB corrupted)
   GainCalibrationFactors* readGainCalibFactors(ULong_t timestamp, const std::map<std::string, std::string>& metadata);
 
-  /// \brief Store Trigger DCS data in the CCDB
-  /// \param dcs trigger DCS data to be stored
+  /// \brief Store FEE DCS data in the CCDB
+  /// \param dcs FEE DCS data to be stored
   /// \param metadata Additional metadata that can be used in the query
   /// \param timestart Start of the time range of the validity of the object
   /// \param timeend End of the time range of the validity of the object
-  void storeTriggerDCSData(TriggerDCS* dcs, const std::map<std::string, std::string>& metadata, ULong_t timestart, ULong_t timeend);
+  void storeFeeDCSData(FeeDCS* dcs, const std::map<std::string, std::string>& metadata, ULong_t timestart, ULong_t timeend);
```

Comment:
What will happen with the trigger DCS DP? I would guess at some point they will have to be added back. Would have been easier in this case to add new functions for the FeeDCS besides the existing ones for the trigger DCS, in case they are anyhow needed both. This interface change makes the macro tests crashing (since in O2 macros are tested at least for whether they compile).

Reply:
TriggerDCS is included in FeeDCS